_episodes/03-configuration.md

```diff
@@ -184,71 +209,131 @@ rootpath:
-> information about ``drs``, you can visit the ESMValTool
-> [documentation](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/quickstart/find_data.html#cmor-drs).
-{: .callout}
-
-> ## Make your own configuration file
+> In this lesson, we will work with data from
+> [CMIP5](https://esgf-node.llnl.gov/projects/cmip5/)
+> and [obs4mips](https://esgf-node.llnl.gov/projects/obs4mips/).
+> How does ESMValTool know the data structure?
 >
-> It is possible to have several configuration files with different purposes,
-> for example: config-user_formalised_runs.yml, config-user_debugging.yml
-{: .callout}
+>> ## Solution
+>>
+>> - Are you working on your own local machine?
+>>   You need to set the `drs` of the data
+>> in the `config-user.yml` file as:
+>>```yaml
+>>   drs:
+>>     CMIP5: default
+>>     obs4mips: default
+>>```
+>>
+>> - Are you working with on a computer cluster like Jasmin or DKRZ?
+>>   Site-specific `drs` of the data are already listed at the end of the
+>> `config-user.yml` file. You need to uncomment the related lines.
+>> For example, on Jasmin:
+>>```yaml
+>>  # Site-specific entries: Jasmin
+>>  # Uncomment the lines below to locate data on JASMIN
+>>  drs:
+>>  #  CMIP6: BADC
+>>    CMIP5: BADC
+>>  #  CMIP3: BADC
+>>  #  CORDEX: BADC
+>>  #  OBS: BADC
+>>  #  OBS6: BADC
+>>    obs4mips: BADC
+>>  #  ana4mips: BADC
+>>```
+>>
+> {: .solution}
+{: .challenge}
 
-> ## Saving preprocessed data
+> ## Explain the default drs (if working on local machine)
 >
-> In the configuration file, which settings are useful to make sure preprocessed
-> data is stored when ESMValTool is run?
+> 1. In the previous exercise, we set the `drs` of CMIP5 data to `default`.
+> Can you explain why?
+>
+> 2. Have a look at the directory structure of the data.
+> There is the folder `Tier1`. What does it mean?
 >
 >> ## Solution
 >>
-> > If the option ``remove_preproc_dir`` is set to ``false``, then the
-> > ``preproc/`` directory contains all the pre-processed data and the
-> > metadata interface files.
-> > If the option ``save_intermediary_cubes`` is set to ``true``
-> > then data will also be saved after each preprocessor step in the folder
-> > ``preproc``. Note that saving all intermediate results to file will result
-> > in a considerable slowdown.
+>> 1. `drs: default` is one way to retrieve data from a ROOT directory that has no DRS-like structure.
+>> ``default`` indicates that all the files are in a folder without any structure.
+>>
+>> 2. Observational data are organized in Tiers depending on their level of public availability.
+>> Therefore the default directory must be structured accordingly with sub-directories
+>> `TierX` e.g. Tier1, Tier2 or Tier3, even when `drs: default`.
+>>
 > {: .solution}
 {: .challenge}
 
+> ## Make your own configuration file
+>
+> It is possible to have several configuration files with different purposes,
+> for example: config-user_formalised_runs.yml, config-user_debugging.yml
```

Comment:
```suggestion
> for example: config-user_formalised_runs.yml, config-user_debugging.yml. In this case, you have to pass the path to the config file you want to use as a command line option when running the ESMValTool.
```
ESMValTool will first look for the default file (first in your ~/.esmvaltool dir, then in the installation dir). Only if you manually specify another path it will use that one. (This has recently changed. Before, you _always_ had to add the path).